src/Libraries/CoreNodes/String.cs

```diff
@@ -38,6 +38,24 @@ public static object ToNumber(string @string)
             throw new ArgumentException(Properties.Resources.StringToNumberInvalidNumberMessage, "str");
         }
 
+        /// <summary>
+        /// Get Number In String
```

Comment:
Would you be more specific about the description? e.g. get first number string from target string, etc or get all of the number string from target string as a list

Reply:
I don't know how many problems like this from user, I have had to write an entire article, you can check it out here
https://chuongmep.com/Number-In-String

Reply:
Has updated to : "Get all of the number strings from the target string as a string"